CONTRIBUTING.rst

```diff
@@ -189,6 +189,13 @@ Running a file of tests::
   app@...:/app$ ./manage.py test pontoon/base/tests/test_views.py
 
 
+Note that currently, we run some tests with `django` and some with `pytest`.
+`make test` runs all of them, but if you want to run just some specific tests,
+and they are using `pytest`, you should run::
+
+  app@...:/app$ pytest path/to/my/test.py
```

Comment:
The same is true for Django tests, right? Should we add a similar command here?

Reply:
Actually, fun fact: the command in the example doesn't work. That's one of the things I'm trying to fix with `pytest` at the moment. Also, given that I still want to get rid of running tests with django, I'd rather not spend too much time on this, and make a better documentation update with my next PR (it's going to be about `pytest`).

Reply:
I use: `pytest -vsk 'test name'`